Cargo.toml

```diff
@@ -27,17 +29,22 @@ async-graphql-poem = "5.0.9"
 async-stl-client = { git = "https://github.com/btpworks/async-stl-sdk" }
 async-stream = "0.3.3"
 async-trait = "0.1.61"
+backoff = { version = "0.4.0", features = ["futures", "tokio"] }
```

Comment:
I was struggling to figure out where you were using this as was interested in this crate.